README.md

```diff
@@ -6,19 +6,24 @@
 
 ## Summary
 
-NumaFlow is a Kubernetes-native platform to run massive parallel data processing or streaming jobs.
+NumaFlow is a Kubernetes-native platform for running massive parallel data processing and streaming jobs.
 
-Each pipeline is specified as a Kubernetes custom resource which consists of one or more source vertices, data processing vertices and sink vertices. Each vertex runs zero or more pods with auto scaling.
+A NumaFlow Pipeline is implemented as a Kubernetes custom resource, and consists of one or more source, data processing, and sink vertices.
 
-NumaFlow targets to achieve `Exactly-Once` semantics, which means from the data from the source vertex to the sink vertex will be processed `Exactly-Once`.
+NumaFlow installs in less than a minute and is eaasier and chaper to run than full-featured stream processing platforms like Flink and Samza.
```

Comment:
we do not support features like reduce which is in Flink or Samza yet. To be politically correct, should we remove `like Flink and Samza`?

Reply:
type `eaasier`

Reply:
Good idea. FYI, that was already there later in the doc.